File: Core/Entities/PatientAppointment.cs
namespace Core.Entities
{
    public class PatientAppointment
    {
        public PatientAppointment() { }

        public PatientAppointment(string id)
        {
            Id = id;
        }

        public string Id { get; set; }
        public List<Appointment> Items { get; set; } = new List<Appointment>();
         public int? AppointmentTypeId { get; set; }
        public string ClientSecret { get; set; }
        public string PaymentIntentId { get; set; }
        public decimal AppointmentTypePrice  { get; set; }
    }
}

File: client/src/app/models/patientAppointment.ts
import * as cuid from "cuid";

export interface Appointment {
    id: number;
    serviceName: string;
    price: number;
    pictureUrl: string;
    capacity: number;
    hospital: string;
    type: string;
}


export interface PatientAppointment {
    id: string;
    items: Appointment[];
    clientSecret?: string;
    paymentIntentId?: string;
    appointmentTypeId?: number;
    appointmentTypePrice:number;
}

export class PatientAppointment implements PatientAppointment {
    id = cuid();
    items: Appointment[] = [];
    appointmentTypePrice=0;
}

export interface PatientAppointmentTotals {
    additionalCosts: number;
    subtotal: number;
    total: number;
}